diff --git a/ray_mlu/mlu.py b/ray_mlu/mlu.py
index 21a57719..2c63fd5b 100755
--- a/ray_mlu/mlu.py
+++ b/ray_mlu/mlu.py
@@ -87,6 +87,3 @@ class MLUAcceleratorManager(AcceleratorManager):
         if os.environ.get(NOSET_MLU_VISIBLE_DEVICES_ENV_VAR):
             return
 
-        os.environ[
-            MLUAcceleratorManager.get_visible_accelerator_ids_env_var()
-        ] = ",".join([str(i) for i in visible_mlu_devices])
